index.test.ts

```diff
@@ -980,5 +985,37 @@ describe("Replicate client", () => {
     });
   });
 
+  describe("webhooks.default.secret.get", () => {
+    test("Calls the correct API route", async () => {
+      nock(BASE_URL).get("/webhooks/default/secret").reply(200, {
+        key: "whsec_5WbX5kEWLlfzsGNjH64I8lOOqUB6e8FH",
+      });
+
+      const secret = await client.webhooks.default.secret.get();
+      expect(secret.key).toBe("whsec_5WbX5kEWLlfzsGNjH64I8lOOqUB6e8FH");
+    });
+
+    test("Can be used to validate webhook", async () => {
+      const secret = "whsec_5WbX5kEWLlfzsGNjH64I8lOOqUB6e8FH";
+
+      const request = new Request("http://test.host/webhook", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          "Webhook-ID": "123",
+          "Webhook-Timestamp": "1707329251",
+          "Webhook-Signature":
+            "v1,nSIlB+PQG9cA4fNl6ec/aOmwxZktHIR5L0ymA5/Cm3E=",
+        },
+        body: `{event:"output",data:"Hello, world!"}`,
+      });
```

Comment:
I used the same values for the [unit test in replicate-go](https://github.com/replicate/replicate-go/pull/40/files#diff-4dc56e64f310cd8c12bb00f75c09e83f5d875563ea9901788e525487dc6788b3R1637-R1653), and the tests pass. So that gives me a bit more confidence. But I'd like something to hedge against me making the same mistake in two different languages 😅